### PR TITLE
chore: update translation files

### PR DESCRIPTION
**Pull request by KazuCocoa**
runs `npm run crowdin-sync` by manual

### CODE REVIEW

File: assets/locales/ru/translation.json [hunk 1]
```diff
@@ -5,11 +5,13 @@
   "ja": "日本語",
   "zh-CN": "中文",
   "ko": "한국어",
-  "hi": "Hindi",
+  "hi": "Хинди",
   "kn": "Kannada",
   "ml-IN": "Malayalam",
-  "pa-IN": "Punjabi",
-  "te": "Telugu",
+  "pa-IN": "Панджаби",
```
Comment (mykola-mokhnach) [inline assets/locales/ru/translation.json]: language names should be either not translated or should be spelled in their own languages

File: assets/locales/zh-CN/translation.json [hunk 2]
```diff
@@ -5,11 +5,13 @@
   "ja": "日本語",
   "zh-CN": "中文",
   "ko": "韩语",
-  "hi": "Hindi",
-  "kn": "Kannada",
-  "ml-IN": "Malayalam",
-  "pa-IN": "Punjabi",
-  "te": "Telugu",
+  "hi": "印度语",
```
Comment (mykola-mokhnach) [inline assets/locales/zh-CN/translation.json]: same as above

File: assets/locales/ru/translation.json (hunk 1 above)
Comment (KazuCocoa) [reply assets/locales/ru/translation.json]: yeah, i'm seeing the crowdin project

File: assets/locales/ja/translation.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
   "Documentation": "ドキュメント",
   "Search Issues": "Issueを検索する",
   "About Appium": "Appiumについて",
-  "showAppInfo": "App Version: {{appVersion}}\nElectron: {{electronVersion}}\nNode.js: {{nodejsVersion}}\n",
+  "showAppInfo": "アプリバージョン: {{appVersion}}\nElectron: {{electronVersion}}\nNode.js: {{nodejsVersion}}\n",
```
Comment (mykola-mokhnach) [inline assets/locales/ja/translation.json]: 😻

**Comment by mykola-mokhnach** [comment]
have you fixed the stuff in Crowdin or just in the local files?

**Comment by KazuCocoa** [comment]
Yes, I've updated Crowdin side

**Pull request merged**